source/includes/_intents.md

```diff
@@ -41,22 +41,102 @@ Please do not share your access tokens with other users or applications and don'
 
 ## Request
 
-Lorem ipsum dolor sit amet, consectetur adipiscing elit. Mauris scelerisque magna eu 
-sapien rutrum euismod. Praesent est urna, hendrerit vitae luctus non, mollis non odio. 
-Suspendisse potenti. Pellentesque pharetra ultrices lectus, sit amet accumsan justo 
-pellentesque pretium. Quisque et mattis leo, ut elementum magna. Praesent at placerat 
-quam. Ut tellus quam, consectetur quis metus sit amet, fermentum suscipit tortor. 
-Donec elit elit, pellentesque a turpis a, porttitor auctor metus. Fusce non rhoncus 
-erat, quis placerat tortor. Morbi molestie porta purus et consequat. Aliquam laoreet 
-at leo fermentum ornare. Fusce ut lobortis tortor, in tincidunt purus.
+Send your request as a JSON with MIME-type of application/json.
+
+> Example Request:
+
+```json
+{
+   "botId":"1a2b3c4d5e6f7g8h9i0j1k2l",
+   "message":"this is a message", 
+   "conversationId":"12345678987654321"
+}
+```
+
+Field | Type  | Required | Description
+---------- | ---------- | ---------- | ------------------------------
+**botId** | string | true | ID of the bot that will sends the reply.<br />You can find the ID of your bot in our dashboard, or talk to your Customer Success Manger.
+**message** | string | true | The text message that the visitor typed into the chat.
+**conversationId** | string | false | An ID used by the chat provider to identify the conversation.<br />The chat provider can decide the format.
```

Comment:
do we need a conversation id for the intents endpoint as well?

Reply:
It is an optional parameter, that is only used to create the session.